content/en/blog/_posts/2024-03-06-unhealthy-pod-eviction-policy-for-pdb-ga.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: blog
+title: "Kubernetes 1.30: Eviction policy for unhealthy pods guarded by PodDisruptionBudgets graduated to stable"
```

Comment:
How about telling a story about protecting your workloads from disruption? That sounds like a more relevant story to tell than the graduation itself.

Reply:
Good idea. Nevertheless the graduation slipped from this release. I will try to revisit it the next one.